.travis.yml

```diff
@@ -5,13 +5,25 @@ dist: trusty
 
 jobs:
   include:
-    - stage: "All"
-      name: "Tests"
+    - name: "C tests"
       install: docker pull holochain/holochain-rust:develop
-      script: . docker/run-test
-      after_success:
+      script:
+        - . docker/run make build test_c_ci
```

Comment:
The C tests need Holochain Rust build first and that is what is happening here. Extracting that and only building it once would be the next optimization I guess..

Reply:
@lucksus aight, i'll look into that :)